libsolidity/ast/Types.h

```diff
@@ -825,6 +829,8 @@ class ArrayType: public ReferenceType
 	/// The offset to advance in storage to move from one array element to the next.
 	unsigned storageStride() const { return isByteArray() ? 1 : m_baseType->storageBytes(); }
 
+	std::vector<Type const*> subtypes() const override { return { m_baseType }; }
```

Comment:
```suggestion
	std::vector<Type const*> subtypes() const override { return {m_baseType}; }
```

There are two more instances of this in the code below.

Reply:
OK